dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -176,6 +177,12 @@ public CultureInfo Culture
         set => this._culture = value ?? CultureInfo.InvariantCulture;
     }
 
+    /// <summary>
+    /// Gets the Json serializer options currently associated with this <see cref="Kernel"/>.
+    /// </summary>
+    [Experimental("SKEXP0001")]
+    public JsonSerializerOptions? SerializerOptions { get; set; }
```

Comment:
Should we have a collection of serializeroptions for different scenarios where its used?

```csharp
public KernelSerializerOptions? SerializerOptions { get; set; }

public KernelSerializerOptions {
    public JsonSerializerOptions Scenario1 { get; set; }
    public JsonSerializerOptions Scenario2 { get; set; }
    ...
}
```

Reply:
Ideally, we should use only one that would cover all scenarios. Over the time, when using the property in different SK components we may identify certain scenarios that need separate {de}serialization options.